tox.ini

```diff
@@ -1,10 +1,17 @@
 [tox]
+skipsdist = True
 envlist = py{27,35}, py{27,35}-marionette, flake8
 
 [testenv]
-passenv = DISPLAY PYTEST_ADDOPTS
+usedevelop = True
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL SAUCELABS_API_KEY SAUCELABS_USERNAME
 deps = -rrequirements/tests.txt
-commands = pytest --ignore tests/marionette {posargs}
+commands = python setup.py develop \
```

Comment:
I've removed python setup.py develop - I can and will address the commands-on-single-line comment; was just following prevailing style in our other repos.